### PR TITLE
Checkpoint

### PR DESCRIPTION
**Pull request by jkjkim**
Finished part 2

### CODE REVIEW

**Comment by yousefh409** [comment]
Nice job so far, both the project and styling look good!

It looks like you finished up to part 2, so complete the rest, and let me know if any problems or questions arise!

**Comment by jkjkim** [comment]
I finished the implementation! :)

File: hknweb/polls/admin.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        (None,               {'fields': ['question_text']}),
+        ('Date information', {'fields': ['pub_date'], 'classes': ['collapse']}),
+    ]
+    inlines = [ChoiceInline]
+    list_display = ('question_text', 'pub_date', 'was_published_recently')
+    list_filter = ['pub_date']
+    search_fields = ['question_text']
+
+
+admin.site.register(Question, QuestionAdmin)
```
Comment (yousefh409) [inline hknweb/polls/admin.py:22]: There should also be a `admin.site.register(Choice)`

**Comment by yousefh409** [comment]
Looks good, nice job!

For future reference, the templates, styles, and so on belonging to the polls project should be placed in the polls folder.

Have a good rest of the semester!

**Pull request closed without merging**